eventually/static/src/containers/topicList/ItemLink.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import Avatar from 'material-ui/Avatar';
+import LibraryBooks from 'material-ui/svg-icons/av/library-books';
+import Code from 'material-ui/svg-icons/action/code';
+import Group from 'material-ui/svg-icons/social/group';
+import FlatButton from 'material-ui/FlatButton';
+import {Card, CardHeader, CardText, CardActions} from 'material-ui/Card';
+import {blue500, yellow600, lime500} from 'material-ui/styles/colors';
+
```

Comment:
add blank line after import(should be 2)

Reply:
done!